content/docs/examples/advanced-gateways/egress-gateway-mtls-origination/index.md

```diff
@@ -6,14 +6,14 @@ keywords: [traffic-management,egress]
 ---
 
 The [Configure an egress gateway](/docs/examples/advanced-gateways/egress-gateway) example describes how to configure
-Istio to direct the egress traffic through a dedicated service called _egress gateway_.
+Istio to direct egress traffic through a dedicated service called _egress gateway_.
 This example shows how to configure an egress gateway to enable mutual TLS for traffic to external services.
 
-To simulate a host outside the Istio service mesh, namely `nginx.example.com`, you deploy an
-[NGINX](https://www.nginx.com/) server in your Kubernetes cluster without injecting an Istio sidecar proxy into the
-server's pod.
-Then you configure an egress gateway to perform mutual TLS with the created NGINX server.
-Finally, you direct the traffic from the application pods inside the mesh to the created server outside the mesh through
+To simulate an actual mTLS external service, you first deploy an [NGINX](https://www.nginx.com) server
```

Comment:
I have doubts about "mTLS external service". It is an external TLS service, you communicate with by **mutual** TLS protocol. Maybe just use "TLS external service" and not "mTLS external service"?

Reply:
How about?

To simulate an actual external service that supports the mTLS protocol, ...

Reply:
This one is OK.